com.unity.ml-agents/Editor/BehaviorParametersEditor.cs

```diff
@@ -147,7 +147,18 @@ void DisplayFailedModelChecks()
                 {
                     if (check != null)
                     {
-                        EditorGUILayout.HelpBox(check, MessageType.Warning);
+                        if (check.CheckType == Inference.CheckType.Info)
```

Comment:
I wanted to have different types of error messages in the inspector. I ended up creating a CheckType enum. 
Maybe it should be `Inference.BarracudaModelParamLoader.CheckType` instead of `Inference.CheckType`?

Reply:
Could make this a switch instead